### PR TITLE
Fix TestUpdateClusterQueue.

### LINKED ISSUE #3066: TestUpdateClusterQueue brittle test
**What would you like to be cleaned**:
in `queue.manager_test.go`, we attempt to create inadmissible workloads, to verify that they are later active when updating a ClusterQueue. Observe that they are never inadmissible, by adding an assertion right after their creation

https://github.com/kubernetes-sigs/kueue/blob/d786072f994668c0240889d93c63f48a620ea6ad/pkg/queue/manager_test.go#L159-L165

```
// also, manager.DumpInadmissible() will be empty
activeWorkloads := manager.Dump()
wantActiveWorkloads := map[string]sets.String{}
if diff := cmp.Diff(wantActiveWorkloads, activeWorkloads); diff != "" {
  t.Errorf("Unexpected active workloads (-want +got):\n%s", diff)
}
```

**Why is this needed**:
Test still passes after deletion of this section:
https://github.com/kubernetes-sigs/kueue/blob/d786072f994668c0240889d93c63f48a620ea6ad/pkg/queue/manager_test.go#L167-L185

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fix `TestUpdateClusterQueue` to allow to create inadmissible workloads.

#### Which issue(s) this PR fixes:

Fixes #3066

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by mimowo — COMMENTED** [review]
LGTM, just small nits
/approve

File: pkg/queue/manager_test.go [hunk 1]
```diff
@@ -165,6 +167,20 @@ func TestUpdateClusterQueue(t *testing.T) {
...
 	}
 
+	// Verify that all workloads are marked as inadmissible after creation.
+	inadmissibleWorkloads := manager.DumpInadmissible()
+	wantInadmissibleWorkloads := map[string][]string{
+		"cq1": {"default/a"},
+		"cq2": {"default/b"},
+	}
+	if diff := cmp.Diff(wantInadmissibleWorkloads, inadmissibleWorkloads); diff != "" {
+		t.Errorf("Unexpected active workloads (-want +got):\n%s", diff)
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply pkg/queue/manager_test.go]: Oh my... I missed that. Thanks!

**Pull request merged**